code/style.css

```diff
@@ -1,3 +1,195 @@
+/* MOBILE */
+
+:root {
+  box-sizing: border-box;
+}
+
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #efefef30;
+  margin: 0;
+  width: 100% fixed;
+  font-family: "Heebo", sans-serif;
+}
+
+.main-content,
+.projectContainer {
+  margin: 0 16px 0 16px;
+}
+
+.navbar {
+  background: #c0716a;
+  display: flex;
+  justify-content: center;
+  padding: 16px;
+}
+
+.github-logo {
+  width: 32px;
+  display: flex;
+  justify-content: center;
+}
+
+.profile-box {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 16px;
+  margin-top: 20px;
+  align-items: center;
+}
+
+.full-name {
+  font-weight: 400;
+  font-size: 26px;
+  line-height: 1.25;
+  color: #1c1b1b;
+  margin-block-end: 0;
+  margin-block-start: 10px;
+}
+
+.username {
+  font-weight: 200;
+  font-size: 20px;
+  color: #1c1b1b;
+  margin-block-start: 7px;
+}
+
+.profile-picture {
+  border-radius: 50%;
+  height: 120px;
+  width: 120px;
+  margin-top: 15px;
+  border: solid 2px rgba(27, 31, 36, 0.15);
+}
+
+.status-box {
+  padding: 4px 0 4px 0;
+  border: 1px solid #d0d7de;
+  border-radius: 6px;
+  display: block;
+  font-size: 16px;
+  color: #57606a;
+  padding: 8px;
+}
+
+.presentation-text {
+  color: #24292f;
+  margin-top: 22px;
+}
+
+.header-project {
+  font-size: 20px;
+  color: #c0716a;
+  text-transform: capitalize;
+}
+
+.chart-container {
+  margin: 16px;
+}
+
+/* Footer styling */
+footer {
+  background: #c0716a;
+  padding: 0.15rem;
+  color: #24292f;
+  text-align: center;
+}
+
+.footer-text {
+  color: #f2e3e1;
+  font-size: 18px;
+  font-weight: 200;
+  margin-block-end: 0;
+}
+
+.fa {
+  width: 30px;
+  text-align: center;
+  text-decoration: none;
+  margin-top: 10px;
+  color: #f2e3e1;
+  padding: 0.3em;
+  margin-block-end: 0;
+}
+
+.fa:hover {
+  color: #ffffff;
+}
+
+.projectContainer {
+  border: 1px solid #d0d7de;
+  border-radius: 5px;
+  margin-bottom: 20px;
+  padding: 10px;
+  background: #c0716a24;
+}
+
+.pullrequestStyle {
+  font-weight: 600;
+  color: #512d1eac;
+  font-size: 15px;
+}
+
+.project-url {
+  color: #c0716a;
+  font-size: 15px;
+}
+
+/* TABLET */
+@media only screen and (min-width: 768px) and (max-width: 1023px) {
```

Comment:
Professional styling! Only one comment about the responsiveness of the page. The "boxes" with information about your repos are placed in one column on mobile size, tablet and desktop. Was it your intention not to place them next to each other on wider screens?

Reply:
Thanks, Ida! I started doing flex and grid for them but didn't have enough time so I skipped it. My intention was to place two or three in a row but didn't manage to fix that. I wasnt sure about if I had to do another container inside innerHTML in JavaScript or how I should do it, i struggled a bit with the structure. Did you create both the parent container and the child container in HTML?

Reply:
In the innerHTML I put a `<div>` and inside that I put a `<form>` with `<ul>` inside. So you should be able to out it all there I think.